_config.yml

```diff
@@ -26,9 +26,9 @@ menu:
           - id: news
             label: 'News / Events'
             url: '/news-events.html'
-          - id: mailinglist
-            label: 'Mailing List'
-            url: '/mailing-list.html'
+          - id: communitycontact
```

Comment:
Could you rename the id "communitycontact[s]" for consistency with label and url?